client/css/style.css

```diff
@@ -176,6 +176,126 @@ button {
 	color: rgba(0, 0, 0, .35) !important;
 }
 
+/* Icons */
+
+#viewport .lt:before,
+#viewport .rt:before,
+#chat button.menu:before,
+#sidebar .chan:before,
+#chat .title:before,
+#footer .icon,
+#chat .count:before,
+#settings #play:before,
+#form #submit:before,
+#chat .invite .from:before,
+#chat .join .from:before,
+#chat .kick .from:before,
+#chat .part .from:before,
+#chat .quit .from:before,
+#chat .topic .from:before,
+#chat .mode .from:before,
+#chat .ctcp .from:before,
+#chat .whois .from:before,
+#chat .nick .from:before,
+#chat .action .from:before,
+.context-menu-item:before {
+	font: normal normal normal 14px/1 FontAwesome;
+	font-size: inherit; /* Can't have font-size inherit on line above, so need to override */
+	-webkit-font-smoothing: antialiased;
```

Comment:
Sadly, we cannot: `subpixel-antialiased` is what is used by default by browsers. So setting it here on icons would kill the point of this PR and go back to ugly rendering :D

Also, note that this is what's used by the Font Awesome CSS, so I'm happy to trust them on this one. As a matter of fact, it's been attempted at different occasions by them or their contributors, see:
- https://github.com/FortAwesome/Font-Awesome/pull/3136#issuecomment-38551961
- https://github.com/FortAwesome/Font-Awesome/issues/1591#issuecomment-21711601
- https://github.com/FortAwesome/Font-Awesome/issues/611
- etc.

Also, this is what [Bootstrap does in their CSS for their icons](https://github.com/twbs/bootstrap/blob/d3cbb8eac266ee21b0e653a49f71007e720ccdb2/dist/css/bootstrap.css#L269-L279).